src/models/cart.js

```diff
@@ -20,4 +20,4 @@ const cartSchema = new schema({
   ],
 });
 
-module.exports = mongoose.model("Cart", cartSchema);
+module.exports = mongoose.model("cart", cartSchema);
```

Comment:
why changed?

Reply:
you told model name should be on small case so

Reply:
I said to maintain consistency because 2 models have uppercase and 2 model has small letter that's should not happened